pkg/cover/html.go

```diff
@@ -18,30 +18,36 @@ import (
 	"sort"
 	"strconv"
 	"strings"
+	"sync"
 
 	"github.com/google/syzkaller/pkg/cover/backend"
 	"github.com/google/syzkaller/pkg/mgrconfig"
 	"github.com/google/syzkaller/sys/targets"
 )
 
 func fixUpPCs(target string, progs []Prog, coverFilter map[uint32]uint32) []Prog {
+	var mu sync.Mutex
```

Comment:
What data is this supposed to protect?
I don't see how a local mutex can protect anything, each invocation of the function uses it's own mutex.

Reply:
when ```f target.Arch == targets.ARM64 && err.Error() == "coverage doesn't match any coverage callbacks" {``` added, there test shows race condition, I guess it's because progs are modified by multiple thread, so that's why I added the mux here.